x-pack/plugins/fleet/cypress/support/commands.ts

```diff
@@ -15,3 +15,27 @@
 // https://on.cypress.io/custom-commands
 // ***********************************************
 //
+/* eslint-disable-next-line import/no-extraneous-dependencies */
+import 'cypress-axe';
+/* eslint-disable-next-line import/no-extraneous-dependencies */
+import { AXE_CONFIG, AXE_OPTIONS } from '@kbn/axe-config';
+
+const axeConfig = {
+  ...AXE_CONFIG,
+};
+const axeOptions = {
+  ...AXE_OPTIONS,
+  runOnly: [...AXE_OPTIONS.runOnly, 'best-practice'],
+};
+
+export const checkA11y = ({ skipFailures }: { skipFailures: boolean }) => {
```

Comment:
it would be nice to add a link to documentation to the [readme](https://github.com/elastic/kibana/blob/main/x-pack/plugins/fleet/cypress/README.md), I mean some context about what are the rules that `checkA11y` checks.

Reply:
Done.